templateChecks.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
Only under certain error. I mean when there's an unexpected error in any operation.